### ISSUE TITLE
Source file missing?

### ISSUE THREAD

**Issue by askew** [Microsoft/service-fabric#135]
I'd like to see why the certificate export for a container doesn't work. (for issue https://github.com/Azure/service-fabric-issues/issues/448)

I can't find the code file for the class header in https://github.com/Microsoft/service-fabric/blob/f258f7579af9643dac6b1c75c93db9a3bcd28fdd/src/prod/src/Common/CertificateManager.h

Is CertificateManager.cpp missing?

**Comment by bpm-ms** [comment]
You're right.  That does seem to be missing from the Github repo.  The repo was trimmed down to code needed to build on linux, while we work on extracting the windows side of the product from internal dependencies.  This was done by recording all of the file reads by CMake during a complete linux build and building a new repository with those files.  The build was then verified via another clean build.  This isn't ideal, but is the compromise we came to while we work on getting the full crossplatform build out there.  

My initial guess is that the linux has an unused header reference, so the header was pulled and not the source file. 

A high priority task for us is to get the windows source out here, and I believe this missing file will come with it.  This is something we're working on making happen as soon as possible since it's a big blocker in our transition to developing out in the open.
